### ISSUE TITLE
[Bug]: iOS - Wrong tx type in asset details and history when user creates a new pool

### ISSUE THREAD

**Issue by ShlyapkinIN** [sora-xor/Sora-iOS#194]
# Preconditions
Blockchain doesn't have a XOR-CERES pool

# Steps to reproduce
1. Create and deposit liquidity to XOR-CERES pool
2. Open Ceres asset details

# Actual result
User sees transfer transaction from tech address

# Expected Result
User sees Create pool tx

**Comment by ShlyapkinIN** [comment]
Can't reproduce

**Comment by Trezeguet1** [comment]
Blocked by https://github.com/sora-xor/sora-ios/issues/219.

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeDl0QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b470239e32cce8d5be5c61537227bb6d8072183c/telegram-cloud-photo-size-2-5314255467788226937-y.jpg)

**Comment by Trezeguet1** [comment]
Verified on Sora Dev 1.0 (238) for iOS version.

User sees transfer transaction from tech address - this is when the user replenishes liquidity in the pool and there are two transfers under the hood - tokens are transferred to technical addresses.

We can open the explorer:
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws.framenode-1.r0.dev.sora2.soramitsu.co.jp#/explorer

We performed the operation in the application and opened our transaction and it will be visible in the events.

<img width="1438" alt="Screenshot 2023-10-31 at 12 04 41" src="https://github.com/sora-xor/sora-ios/assets/51509139/0261d58c-f207-4db9-b6fc-2ec244d89931">
